ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -756,6 +807,15 @@ literals:
 - position_status
 - condensing_water_valve_percentage_sensor
 - entering_cooling_coil_temperature_sensor
+- condensing_supply_water_isolation_valve_open_status
```

Comment:
does the device have a singulary valve status that provides OPEN/CLOSED as its states that can be used in place of these?